python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -1013,6 +1013,42 @@ def test_stat_cov(self):
             self.spark.read.table(self.tbl_name2).stat.cov("col1", "col3"),
         )
 
+    def test_stat_approx_quantile(self):
+        # SPARK-41069: Test the stat.approxQuantile method
+        result = self.connect.read.table(self.tbl_name2).stat.approxQuantile(
```

Comment:
let's compare the results with PySpark

Reply:
The result is different between each execution.